src/test/groovy/graphql/schema/SchemaTraverserTest.groovy

```diff
@@ -388,15 +382,6 @@ class SchemaTraverserTest extends Specification {
 
     }
 
-
```

Comment:
These are old tests, a `typeResolver` is no longer required on the union/interface type